plugins/pulp_rpm/plugins/distributors/yum/metadata/metadata.py

```diff
@@ -201,54 +201,3 @@ def _is_closed(file_object):
                 return file_object.myfileobj is None or file_object.myfileobj.closed
             else:
                 raise
-
-
-# -- pre-generated metadata context --------------------------------------------
-
-class PreGeneratedMetadataContext(MetadataFileContext):
```

Comment:
It's a dead code, I could not find any usage of it.
It was deleted in the scope of this PR because `PreGeneratedMetadataContext` use `repodata` attribute which I had to  fix in case I left this code.